examples/opentracing-shim/README.md

```diff
@@ -0,0 +1,75 @@
+# Overview
+
+OpenTelemetry Shim Opentracing allows user to report to OpenTelemetry with existing OpenTracing instrumentation.
+
+This is a simple example that demonstrates tracing HTTP from client to server.
+
+The example shows key aspects of tracing such as
+
+- Root Span (on client)
```

Comment:
should add how it works with OpenTracing shim

Reply:
You could add:

`- Make a shim between OpenTracing and OpenTelemetry tracers`

WDYT ?